doc/UsersGuide/source/tables/fv3diagtable.csv

```diff
@@ -0,0 +1,416 @@
+uswsfci,Instantaneous surface upward shortwave flux
+dusfci,Instantaneous u component of surface stress
+dvsfci,Instantaneous v component of surface stress
+shtfl,Instantaneous surface sensible heat net flux
+lhtfl,Instantaneous surface latent heat net flux
+gfluxi,Instantaneous surface ground heat flux
+wilt,Wiltimg point (volumetric)
+fldcp,Field capacity (volumetric)
+pahi,Instantaneous precipitation advected heat flux
+pevpr,Instantaneous surface potential evaporation
+wet1,Normalized soil wetness
+cpofp,Precent frozen precipitation
+crain_ave,Averaged categorical rain
+csnow_ave,Averaged categorical snow
+cfrzr_ave,Averaged categorical freezing rain
+cicep_ave,Averaged categorical sleet
+refl_10cm,Radar reflectivity
+max_hail_diam_sfc,Maximum hail diameter at lowest model level
+dkt,Atmospheric heat diffusivity
+dku,Atmospheric momentum diffusivity
+cldfra,Instantaneous 3D cloud fraction
+cnvw,Subgrid scale convective cloud water
+skebu_wts,Perturbation velocity - u component
+skebv_wts,Perturbation velocity - v component
+zmtnblck,Level of dividing streamline
+refdmax,Max hourly 1-kilometer above ground level reflectivity
+refdmax263k,Max hourly -10C reflectivity
+sppt_wts,Perturbation velocity
+shum_wts,Perturbation velocity
+spp_wts_pbl,Stochastic perturbed parameterization planetary boundary layer perturbation wind turbines
+spp_wts_pbl,Stochastic perturbed parameterization surface perturbation wind turbines
+spp_wts_mp,Stochastic perturbed parameterization Moller-Plesset perturbation wind turbines
+spp_wts_gwd,Stochastic perturbed parameterization gravitational waves perturbation wind turbines
+spp_wts_rad,Stochastic perturbed parameterization radial perturbation wind turbines
+spp_wts_cu_deep,Stochastic perturbed parameterization cu deep perturbation wts
+sfc_wts,Perturbation amplitude
+ca1,Cellular automata
+ca_deep,Cellular automaton deep convection
+ca_turb,Cellular automaton turbulence
+ca_shal,Cellular automaton shallow convection
+ca_rad,Cellular automaton radiation
+ca_micro,Cellular automaton microphysics
+lakefrac,Lake fraction
+lakedepth,Lake depth
+T_snow,Temperature of snow on a lake
+T_ice,Temperature of ice on a lake
+use_lake_model,Lake model flag
+lake_is_salty,Lake point is considered salty by CLM lake model
+lake_cannot_freeze,CLM lake model considers the point to be so salty it cannot freeze
+lake_t2m,Temperature at 2 meters from lake model
+lake_q2m,2m specific humidity from lake model
+lake_albedo,Mid-day surface albedo over lake
+lake_h2osno2d,Water equivalent of accumulated snow depth over lake
+lake_sndpth2d,Actual accumulated snow depth over lake in CLM lake model
+lake_snl2d,Snow layers in CLM lake model (treated as integer)
+lake_tsfc,Skin temperature from CLM lake model
+lake_savedtke12d,Top level eddy conductivity from previous timestep in CLM lake model
+lake_ht,Lake height
+zmtb,Height of dividing streamline
+zogw,Height of orographic gravity waves
+zlwb,Height of lamb wave beam

```

Comment:
```suggestion
zlwb,Height of low-level wave breaking
```

Reply:
@selbigmtnwx23 FYI, I will adjust some of the abbreviations in this section based on what I'm seeing in the CCPP CIRES Gravity Wave Drag module: https://dtcenter.ucar.edu/GMTB/v6.0.0/sci_doc/group__cires__ugwp__run__mod_gabfb96aeb18e1151f7804e868975c9a9d.html#gabfb96aeb18e1151f7804e868975c9a9d